python/mxnet/optimizer/optimizer.py

```diff
@@ -1244,6 +1244,51 @@ def update(self, index, weight, grad, state):
             kwargs = {}
             sgd_update(weight, grad, out=weight, lr=lr, wd=wd, **kwargs)
 
+
+@register
+class LAMB(Optimizer):
+    """LAMB Optimizer.
```

Comment:
pls add doc

Reply:
working on it now

Reply:
The name is clashing with the GLuon one, can we give it a different name?